clang/lib/CIR/CodeGen/CIRGenStmt.cpp

```diff
@@ -487,7 +487,7 @@ mlir::LogicalResult CIRGenFunction::buildReturnStmt(const ReturnStmt &S) {
       // Make sure not to return anything, but evaluate the expression
       // for side effects.
       if (RV) {
-        assert(0 && "not implemented");
+        buildScalarExpr(RV);
```

Comment:
why not `buildAnyExpr`?

Reply:
Ahh don't know about this one. Thanks for pointing out! Fixed.